websites/api-docs-smoke-test/gatsby-config.js

```diff
@@ -5,6 +5,13 @@ const {
 const colorPresets = require('@commercetools-docs/gatsby-theme-docs/color-presets');
 
 module.exports = {
+  // https://www.gatsbyjs.com/docs/reference/release-notes/v2.28/#feature-flags-in-gatsby-configjs
+  // https://www.gatsbyjs.com/docs/reference/release-notes/v2.30
+  flags: {
+    DEV_SSR: true,
+    FAST_REFRESH: true,
+    PRESERVE_WEBPACK_CACHE: true,
```

Comment:
Is this experimental flag (PRESERVE_WEBPACK_CACHE) intended for our production sites too?

Reply:
I'm not sure I understand the question 🤔

Reply:
> I'm not sure I understand the question 🤔

From what I read on Gatsby's website, PRESERVE_WEBPACK_CACHE flag is experimental. So my questions is do we intend to use this flag on sites in our commercetools-docs repo?

Link - https://www.gatsbyjs.com/docs/reference/release-notes/v2.28/#experimental-new-cache-clearing-behaviors

Reply:
I think we should yes.

But still, the changes here are unrelated to that. You have to specify the flags for each website anyway.

Reply:
> I think we should yes.

I would be wary of using anything experimental in production.

Reply:
I'm not sure where do you get this idea, but the flags are for development.